pkg/sql/parser/sql.y

```diff
@@ -1595,6 +1597,26 @@ import_stmt:
   }
 | IMPORT error // SHOW HELP: IMPORT
 
+
+// %Help: EXPORT - export data to file in a distributed manner
+// %Category: CCL
+// %Text:
+// EXPORT <format> (<datafile> [WITH <option> [= value] [,...]]) <query>
+//
+// Formats:
+//    CSV
+//
+// Options:
+//    comma = '...'          [CSV-specific]
+//
+// %SeeAlso: SELECT
+export_stmt:
+  EXPORT import_data_format '(' string_or_placeholder opt_with_options ')' select_stmt
```

Comment:
the WITH outside of the parentheses?

Reply:
whoops, just commented top-level about that, but i got a conflict with it immediately next to `select_stmt`. I'm also not completely sure about the `()` for that matter.